sydent/sms/openmarket.py

```diff
@@ -61,8 +60,7 @@ def __init__(self, sydent: "Sydent") -> None:
         self.sydent = sydent
         self.http_cli = SimpleHttpClient(sydent)
 
-    @defer.inlineCallbacks
-    def sendTextSMS(
+    async def sendTextSMS(
         self, body: Dict, dest: str, source: Optional[Dict[str, str]] = None
     ) -> Generator:
```

Comment:
This no longer returns a Generator.

Reply:
I removed `Generator` and added `NoReturn`, as I didn't see a `return` anywhere in the function, and it resulted in the same mypy error as the comment above: `misc - Implicit return in function which does not return`. I'm not entirely sure what the error is referring to, any insight would be appreciated!